### PR TITLE
Changed chromedriver to geckodriver

### PR DESCRIPTION
**Pull request by AnnaLangner**
I did it: `Modify script test_2 so it will use Firefox driver instead of chrome.` but it won't work without sleeps :( I try to change sleeps to explicit wait.

### CODE REVIEW

File: script/test_2.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 def main():
-  driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  driver = webdriver.Firefox(executable_path="bin/geckodriver.exe")
+  # driver = webdriver.Chrome(executable_path="bin/chromedriver")
   driver.get("https://en.wikipedia.org/wiki/Main_Page")
   search_input = driver.find_element_by_id("searchInput")
   search_input.clear()
   search_input.send_keys("Unit testing")
   search_input.submit()
+  time.sleep(3)
```
Comment (klangner) [inline script/test_2.py]: Try to replace sleep with WebDriver Waits.

**Review by klangner — CHANGES_REQUESTED** [review]

**Review by klangner — APPROVED** [review]

File: script/test_2.py [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
   
 
 def main():
-  driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--driver', help='entry driver')
+  firefox = webdriver.Firefox(executable_path="bin/geckodriver.exe")
```
Comment (klangner) [inline script/test_2.py]: Do not create drivers here

File: script/test_2.py [hunk 3]
```diff
@@ -1,17 +1,36 @@
...
+  chrome = webdriver.Chrome(executable_path="bin/chromedriver")
+  args = parser.parse_args() 
+  driver_choice = args.driver  
+  if driver_choice == 'chrome':
+    driver = chrome
```
Comment (klangner) [inline script/test_2.py]: Create driver here

File: script/test_2.py [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
+  elif driver_choice == 'firefox':
```
Comment (klangner) [inline script/test_2.py]: this elif is not necessary.

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/test_2.py [hunk 5]
```diff
@@ -1,17 +1,32 @@
...
+  if driver_choice == 'chrome':
+    driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  else:
+    driver = webdriver.Firefox(executable_path="bin/geckodriver.exe")
   driver.get("https://en.wikipedia.org/wiki/Main_Page")
   search_input = driver.find_element_by_id("searchInput")
   search_input.clear()
   search_input.send_keys("Unit testing")
   search_input.submit()
+  time.sleep(3)
```
Comment (klangner) [inline script/test_2.py]: Why sleep instead of WebDriverWait?

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/test_2.py (hunk 5 above)
Comment (AnnaLangner) [reply script/test_2.py]: When I run the script from --driver chrome, I get a WebDriverWait error :
```
$ python script/test_2.py --driver "chrome"

DevTools listening on ws://127.0.0.1:60079/devtools/browser/36b29f9e-238f-4919-a5ab-1a9cb3c51db0
Traceback (most recent call last):
  File "script/test_2.py", line 39, in <module>
    main()
  File "script/test_2.py", line 27, in main
    WebDriverWait(driver, 10).until(EC.url_changes(driver.current_url))
  File "C:\Users\annal\AppData\Local\Programs\Python\Python38-32\lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until     
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message:
```

Comment (klangner) [reply script/test_2.py]: Try to wait for something else

**Review by klangner — CHANGES_REQUESTED** [review]

**Review by klangner — APPROVED** [review]

**Pull request merged**